hathor/transaction/scripts/sighash.py

```diff
@@ -60,7 +60,37 @@ def _get_indexes(bitmask: int) -> list[int]:
         return [index for index in range(8) if (bitmask >> index) & 1]
 
 
-SighashType: TypeAlias = SighashAll | SighashBitmask
+class SighashRange(CustomSighash):
+    """A model representing the sighash range type config. Range ends are not inclusive."""
```

Comment:
Shouldn't range ends be included? I feel it's more intuitive than not including them.